hydra-core/spec/controllers/catalog_controller_spec.rb

```diff
@@ -42,6 +42,13 @@
   end
 
   describe "content negotiation" do
+    before do
```

Comment:
@jrgriffiniii Can you please explain why these routes need to be added here?  I couldn't get to the references you were pointing to.